src/super_gradients/training/models/detection_models/yolo_base.py

```diff
@@ -80,11 +80,14 @@ def __init__(
         self.with_confidence = with_confidence
 
     def forward(self, x, device: str = None):
```

Comment:
Could you please add type hints to x to annotate what are the supported input types will be?

Reply:
added type hints [type hints](https://github.com/Deci-AI/super-gradients/pull/813/commits/dab2d28e2e57b75198d48822deebd94ca197301a)